src/test/java/io/stargate/sgv2/jsonapi/api/model/command/impl/FindCommandTest.java

```diff
@@ -162,6 +162,30 @@ public void invalidOptionsSkipNoSort() throws Exception {
           .contains("skip options should be used with sort clause");
     }
 
+    @Test
+    public void invalidOptionsSkipEmptySort() throws Exception {
+      String json =
+          """
+              {
+              "find": {
+                  "filter" : {"username" : "user1"},
+                  "sort": {},
+                  "options" : {
+                    "skip" : 10
+                  }
+                }
+              }
+              """;
+
+      FindCommand command = objectMapper.readValue(json, FindCommand.class);
+      Set<ConstraintViolation<FindCommand>> result = validator.validate(command);
+
+      assertThat(result)
+          .isNotEmpty()
+          .extracting(ConstraintViolation::getMessage)
+          .contains("skip options should be used with sort clause");
```

Comment:
Minor nitpick that I know wasn't introduced with this PR, but can we change "skip options" -> "skip option"? There's only one skip option.

Reply:
Good catch, thanks!